recipe/meta.yaml

```diff
@@ -37,12 +37,17 @@ test:
   imports:
     - psycopg2
     - psycopg2._psycopg
+  commands:
+    - pip check
+  requires:
+    - pip
 
 about:
   home: https://www.psycopg.org/
   license: LGPL-3.0-or-later AND Other
```

Comment:
The source code is under `LGPL-3.0-or-later` (see https://www.psycopg.org/: `Psycopg is released under the terms of the [GNU Lesser General Public License](https://www.psycopg.org/license/), allowing use from both free and proprietary software.`) but the `BSD-like` license applies (at your option) to the files following the pattern ``psycopg/adapter*.{h,c}`` and ``psycopg/microprotocol*.{h,c}``.
Textually this BSD-like license is identical only to `zlib` license https://spdx.org/licenses/Zlib.html.
We can say that the license is `LGPL-3.0-or-later` because the `BSD-like` is optional.